src/node_report_utils.cc

```diff
@@ -0,0 +1,306 @@
+#include "node_report.h"
+#include "node_internals.h"
+#include "node_options.h"
+#include "env.h"
+#include <v8.h>
+#include "node_report.h"
+#include "node_internals.h"
+#include "node_options.h"
+#include "env.h"
+#include "v8.h"
+#include "util.h"
+
+#if HAVE_OPENSSL
+#include "node_crypto.h"
+#endif
+
+
+#ifdef __APPLE__
+#include <crt_externs.h>  // _NSGetArgv() and _NSGetArgc()
+#endif
+#ifdef __sun
+#include <procfs.h>  // psinfo_t structure
+#endif
+#ifdef _AIX
+#include <sys/procfs.h>  // psinfo_t structure
+#endif
```

Comment:
None of `_NSGetArgc`, `_NSGetArgv` or `psinfo_t` actually appear anywhere in the source code besides here… can these be removed?

Reply:
removed